x-pack/platform/plugins/shared/observability_ai_assistant/server/service/client/operators/get_generated_title.ts

```diff
@@ -13,7 +13,8 @@ import { Message, MessageRole } from '../../../../common';
 import { LangTracer } from '../instrumentation/lang_tracer';
```

Comment:
Shouldn't this file be in https://github.com/elastic/kibana/tree/main/x-pack/platform/plugins/shared/observability_ai_assistant/server/functions ?

Reply:
Same with the score function:

https://github.com/elastic/kibana/blob/81f69713f37dc4f355744327ee9ae84a54d9f147/x-pack/platform/plugins/shared/observability_ai_assistant/server/utils/recall/score_suggestions.ts#L93-L122